office_space_allocation/amity.py

```diff
@@ -0,0 +1,150 @@
+import random
+
+from office_space_allocation.fellow import Fellow
+from office_space_allocation.office import Office
+from office_space_allocation.person import Person
+from office_space_allocation.staff import Staff
+from office_space_allocation.utilities import InvalidRoomOccupantError, RoomFullError, MultiplePeopleFoundException
+
+
+class Amity:
+    def __init__(self):
+        self.all_persons = []
+        self.all_rooms = []
+        self.allocated_rooms = []
+
+    def add_person(self, person):
+        """
+        Adds a new Person to list of persons
+        :param: ```Person``` person
+        """
+        if not isinstance(person, Person):
+            raise TypeError("Argument should be of type Person")
+        else:
+            self.all_persons.append(person)
+
+    def add_room(self, new_room):
+        """
+        Adds ```Room``` new_room to list of all_rooms
```

Comment:
Your doc strings are a bit confusing. They don't really bring out what the function does